xcode/ios/MoaiSample.xcodeproj/project.pbxproj

```diff
@@ -505,33 +564,41 @@
 		1D6058940D05DD3E006BFB54 /* Debug */ = {
 			isa = XCBuildConfiguration;
 			buildSettings = {
+				ALWAYS_SEARCH_USER_PATHS = NO;
+				DEVELOPMENT_TEAM = JG69W4K637;
```

Comment:
Is this development team id for a personal account? If so we might want to unset that.